clr/config.py

```diff
@@ -37,4 +41,6 @@ def read_namespaces():
     'commands'. The values of this mapping are python module names that *are* on
     the PYTHONPATH and can be imported with importlib.import_module.
     """
-    return runpy.run_path(find_clrfile())['commands']
+    clrfile = find_clrfile()
+    if not clrfile: return {}
```

Comment:
Break into two lines please (should we just add `black` to this repo?) Would mean you don't need to bother with these things. :smile:

Reply:
Done. I have a follow up pr that runs black from the color venv on this whole codebase. Integrated would be nice too.